package.json

```diff
@@ -53,6 +54,14 @@
 		"@rollup/plugin-node-resolve": "^15.2.1",
 		"@rollup/plugin-typescript": "^11.1.2",
 		"@rollup/plugin-url": "^8.0.1",
+		"@storybook/addon-essentials": "^8.0.4",
+		"@storybook/addon-interactions": "^8.0.4",
+		"@storybook/addon-themes": "^8.0.4",
+		"@storybook/addon-webpack5-compiler-babel": "^3.0.3",
+		"@storybook/blocks": "^8.0.4",
+		"@storybook/react": "^8.0.4",
+		"@storybook/react-webpack5": "^8.0.4",
+		"@storybook/test": "^8.0.4",
```

Comment:
cleanup also the deps with only the ones needed with the current configuration

Reply:
I updated the `package.json`, I think the rest are used or required